script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_10B.txt

```diff
@@ -26,8 +26,8 @@ Hey, Ando!"
 "Hey, did you call me, young master?
 Man, today is not my lucky day. It'd be far-fetched to just write this off as some kind of gas explosion. Should we get some terrorist organization to make a statement?"
 <Page8>
-"No need for that. We'll leave the external affairs to the owner of this area. After all, he'll start boasting if we hit him enough.
-Our job is to retrieve the things that are not supposed to be here. We've wiped out most traces of the Dead Apostles, but there's still work to be done handling the victims. I'll leave it to you."//ruby dots <こ|・><こ|・><の|・><持|・><ち|・><主|・>
+"No need for that. We'll leave the external affairs to the owner of this area. After all, he'll start boasting if we hit him enough.//is that really the meaning of 叩けばホコリが出る?
```

Comment:
I think this is like a pinata kinda. "If We hit him [enough], his pride will come out"